### ISSUE TITLE
TypeError: _this5[_symbols2.default.self].flexData[Symbol.iterator] is not a function

### ISSUE THREAD

**Issue by Gert-dev** [twifty/atom-phpunit-integrator#40]
```
TypeError: _this5[_symbols2.default.self].flexData[Symbol.iterator] is not a function
    at previousFlex (/home/user/.atom/packages/php-unit-integrator/node_modules/colletch/components/etch-flex-container.js:335:58)
    at /home/user/.atom/packages/php-unit-integrator/node_modules/colletch/components/etch-flex-container.js:353:23
    at Array.map (native)
    at EtchFlexContainer.value (/home/user/.atom/packages/php-unit-integrator/node_modules/colletch/components/etch-flex-container.js:345:58)
    at ResizeObserver.<anonymous> (/home/user/.atom/packages/php-unit-integrator/node_modules/colletch/components/etch-flex-container.js:85:64)
```

Not exactly sure why or how this started happening, but all of a sudden I can't avoid getting this error anymore. Every start up generates multiple of them.

Toggling the console for some reason is duplicating the two toggle buttons `Debug` and `Code Coverage` as well, which is probably related.

**Comment by twifty** [comment]
I'm currently working on the issue. The problem comes from serializing the panel sizes and adding/removing a new panel. A lot of floating point math involved which makes my head hurt.

**Comment by twifty** [comment]
I think this is fixed in 0.0.43. If the problem persists, or you notice any sizing issues with the tree/console/coverage panels, please reopen.
